python/docs/support/usage_statistics.rst

```diff
@@ -1,17 +1,22 @@
 Usage Statistics
 ================
 
-Describe our story about usage statistics
+whylogs collects anonymous information about a user’s environment. This includes information such as:
 
-Why Statistics?
-################
+whylogs version
+Operating system
+Python version
+Execution environment (Sagemaker, Google Colab, Jupyter Notebook, etc.)
+Relevant libraries in the runtime environment (numpy, pandas, etc.)
+This data helps our developers to deliver the best possible product to our users. This information can be used to help inform our team of the best areas to focus development and better understand how our users are utilize whylogs.
 
-.. code-block:: bash
+Data is only collected at the time of importing the whylogs library or its modules. We do not collect any sensitive information, or user code.
 
-   pip install whylogs
+If users wish to opt out of this usage statistics collection, they can do so by setting the WHYLOGS_NO_ANALYTICS environment variable as follows:
```

Comment:
i would make the env var with the correct "variable syntax" (like `this`)

Reply:
or in rst should be :code:`WHYLOGS_NO_ANALYTICS`

Reply:
Good call